src/hooks/useAuth.ts

```diff
@@ -12,13 +12,13 @@ export default function useAuth() {
     refetch,
     isLoading,
   } = useQuery({
-    queryKey: ['user'],
-    queryFn: fetchMyData,
+    queryKey: myQueries.getQueryKey(),
+    queryFn: myQueries.queryOptions().queryFn,
     enabled: !!accessToken,
```

Comment:
❗`myQueries`의 `queryOptions`가 `queryKey`랑 `queryFn`을 반환하고 있으니까 스프레드로 풀어서 사용하면 어떨까 싶어요!

```suggestion
    ...myQueries.queryOptions(),
    enabled: !!accessToken,
```

Reply:
네 수정했습니다.